addon/components/basic-dropdown-content.js

```diff
@@ -177,7 +177,15 @@ export default @layout(templateLayout) @tagName('')class BasicDropdownContent ex
   @action
   setupMutationObserver(dropdownElement) {
     this.mutationObserver = new MutationObserver((mutations) => {
-      if (mutations[0].addedNodes.length || mutations[0].removedNodes.length) {
+      let addedNodes = Array.prototype.slice.call(mutations[0].addedNodes).filter((node) => {
```

Comment:
Use `Array#some` instead of `Array#filter` and then length.

Reply:
Sounds good. updated. 👍